ops/ccs-ops-misc/load_test/common/url_path.py

```diff
@@ -0,0 +1,13 @@
+import time
```

Comment:
Is this being used?

Reply:
Good catch! I must have accidentally imported that automatically when writing this, I'll remove it.